frontend/config.js

```diff
@@ -3,6 +3,11 @@ module.exports = {
   IS_DEBUG: (process.env.NODE_ENV === 'development'),
   USE_HTTPS: (process.env.USE_HTTPS == 1),
   ENABLE_PONTOON: (process.env.ENABLE_PONTOON === '1'),
+  AVAILABLE_LOCALES: (process.env.ENABLE_DEV_LOCALES === '1') ?
+    // All locales on Pontoon for local & dev
+    'en-US,zh-CN,de,ja,fy-NL,fa,zh-TW,ast,uk,fr,sv-SE,bn-BD,pt-BR,tr,kab,es-ES,sl,cs,ru,bs,es-CL' :
```

Comment:
it'd be cool if we could automate this all locales list. maybe with a script that gets the directory names under `/locales`